src/main/java/seedu/address/model/statistics/exceptions/DuplicateDataException.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.model.statistics.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate StatisticData
+ *  (StatisticData are considered duplicates if they have the same name).
```

Comment:
there's an extra space here
```suggestion
 * (StatisticData are considered duplicates if they have the same name).
```

Reply:
ok